cmd/main.go

```diff
@@ -80,135 +58,13 @@ func main() {
 		panic("error, variable customEntriesFormat is not set")
 	}
 
-	mat, err := commatrix.New(kubeconfig, customEntriesPath, customEntriesFormat, env, deployment)
-	if err != nil {
-		panic(fmt.Sprintf("failed to create the communication matrix: %s", err))
-	}
-
-	res, err := printFn(*mat)
-	if err != nil {
-		panic(err)
-	}
-
-	comMatrixFileName := filepath.Join(destDir, fmt.Sprintf("communication-matrix.%s", format))
-	err = os.WriteFile(comMatrixFileName, res, 0644)
-	if err != nil {
-		panic(err)
-	}
-
-	cs, err := clientutil.New(kubeconfig)
-	if err != nil {
-		panic(err)
-	}
-
-	tcpFile, err := os.OpenFile(path.Join(destDir, "raw-ss-tcp"), os.O_TRUNC|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		panic(err)
-	}
-	defer tcpFile.Close()
-
-	udpFile, err := os.OpenFile(path.Join(destDir, "raw-ss-udp"), os.O_TRUNC|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		panic(err)
-	}
-	defer udpFile.Close()
-
-	nodesList, err := cs.CoreV1Interface.Nodes().List(context.TODO(), metav1.ListOptions{})
+	ssComMat, endSliceMat, err := commatrix.GenerateMatrix(kubeconfig, customEntriesPath, customEntriesFormat, format, env, deployment, destDir)
```

Comment:
GenerateMatrix  returns first the communication matrix only then the ss matrix:
`return mat, ssComMat, err`